src/styles/main.scss

```
@@ -0,0 +1,8 @@
+@import "./units/_variables.sass";
```

why are you using a different file extension here?
```suggestion
@import "./units/_variables.sсss";
```